php-practice_1.php

```diff
@@ -1,41 +1,118 @@
 <?php
 // Q1 変数と文字列
-
+$name = '西原';
+echo $name = ' 私の名前は 「' . $name . '」です。 ';
 
 // Q2 四則演算
-
+$num = 20;
+echo $num;
+echo $num/2;
 
 // Q3 日付操作
-
+$time = date('Y年m月d日 H時i分s秒');
+echo '現在時刻は、' . $time . 'です。';
 
 // Q4 条件分岐-1 if文
-
+ $device = 'abc';
+if ($device === 'mac' || $device === 'windows') {
+echo '使用OSは、'.$device.'です。';
+}else{
+   echo 'どちらでもありません。';
+}
```

Comment:
[must]
インデントがずれているので修正お願いします。
`.`の前後は半角スペースを設けてください。
```suggestion
if ($device === 'mac' || $device === 'windows') {
    echo '使用OSは、' . $device . 'です。';
} else {
    echo 'どちらでもありません。';
}
```

Reply:
インデント修正・半角スペース修正しました。